configs/orchestrator/templates/cloud-secrets.yaml

```diff
@@ -4,36 +4,53 @@ metadata:
   name: cloud-secrets
   namespace: orchestrator
 data:
+  # EM specific values.
   {{if .Values.clouds.em.token}}
   EQUINIX_METAL_AUTH_TOKEN: {{ .Values.clouds.em.token | b64enc }}
   {{end}}
   {{if .Values.clouds.em.projectID}}
   EQUINIX_METAL_PROJECT_ID: {{ .Values.clouds.em.projectID | b64enc }}
   {{end}}
+
+  # AWS specific values.
   {{if .Values.clouds.aws.accessKeyID}}
   AWS_ACCESS_KEY_ID: {{ .Values.clouds.aws.accessKeyID | b64enc }}
   {{end}}
   {{if .Values.clouds.aws.secretAccessKey}}
   AWS_SECRET_ACCESS_KEY: {{ .Values.clouds.aws.secretAccessKey | b64enc }}
   {{end}}
-  {{if .Values.clouds.aws.defaultRegion}}
-  AWS_DEFAULT_REGION: {{ .Values.clouds.aws.defaultRegion | b64enc }}
-  {{end}}
   {{if .Values.clouds.aws.route53Zone}}
   AWS_ROUTE53_ZONE: {{ .Values.clouds.aws.route53Zone | b64enc }}
   {{end}}
   {{if .Values.clouds.aws.route53ZoneID}}
   AWS_ROUTE53_ZONE_ID: {{ .Values.clouds.aws.route53ZoneID | b64enc }}
   {{end}}
-  {{if .Values.clouds.aws.benchmarkWorkerCount }}
-  BENCHMARK_WORKER_COUNT: {{ .Values.clouds.aws.benchmarkWorkerCount | quote | b64enc }}
+
+  # OCI specific values.
+  {{if .Values.clouds.oci.userOCID }}
+  USER_OCID: {{ .Values.clouds.oci.userOCID | b64enc}}
+  {{end}}
+  {{if .Values.clouds.oci.fingerprint }}
+  FINGERPRINT: {{ .Values.clouds.oci.fingerprint | b64enc}}
+  {{end}}
+  {{if .Values.clouds.oci.tenancyID }}
+  TENANCY_ID: {{ .Values.clouds.oci.tenancyID | b64enc}}
+  {{end}}
+  {{if .Values.clouds.oci.compartmentID }}
+  COMPARTMENT_ID: {{ .Values.clouds.oci.compartmentID | b64enc}}
+  {{end}}
+  {{if .Values.clouds.oci.adNumber }}
+  AD_NUMBER: {{ .Values.clouds.oci.adNumber | b64enc}}
```

Comment:
Actually this is not a secret, so moved it to configmap. Also added a function `quote`, so no matter what format the user provides this information it will be quoted.

As a helm chart author we need to make sure that this entry is quoted because it is embedded into a secret/configmap and the data type accepted there is key value pair of strings. Anything else will fail to deploy.